server/data/mappers/prisonerSummaryMapper.test.ts

```diff
@@ -0,0 +1,78 @@
+import moment from 'moment'
+import type { Prisoner } from 'prisonRegisterApiClient'
+import type { PrisonerSummary } from 'viewModels'
+import aValidPrisoner from '../../testsupport/prisonerTestDataBuilder'
+import aValidPrisonerSummary from '../../testsupport/prisonerSummaryTestDataBuilder'
+import toPrisonerSummary from './prisonerSummaryMapper'
+
+describe('prisonerSummaryMapper', () => {
```

Comment:
I've written a new mapper that maps from `Prisoner` to `PrisonerSummary` (previously we did it inline in `routerRequestHandlers.ts`
These are the tests for the new mapper 👍